cvat-ui/src/components/common/cvat-logo.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+function CVATLogo(): JSX.Element {
+    const SVGLogo = useSelector((state: any) => state.about.server.logo);
+
+    return (
+        <div className='cvat-logo-icon'>
+            <img
```

Comment:
What if the server returns logo (let's say 1024x256px)? Will it be fit into the component?

Reply:
Changed styles a bit. By default logo component is `64x32` so everything bigger will be scaled using `object-fit: contain` to maintain the size